doc/yig_deploy_cn.md

```diff
@@ -0,0 +1,524 @@
+# YIG部署
+
+YIG涉及的组件包括tidb, redis, kafka,caddy, ceph, yig
+
+### 1.关闭防火墙和selinux
+
+```
+systemctl stop firewalld
+systemctl disable firewalld
+setenforce 0
+sed -i '/SELINUX/s/enforcing/disabled/' /etc/selinux/config
+```
+
+### 2.修改主机名
+
+各个机器下分别执行如下命令
+
+```
+hostnamectl set-hostname  ceph117
+hostnamectl set-hostname  ceph118
+hostnamectl set-hostname  ceph119
+```
+
+修改后在每台机器上修改/etc/hosts文件
+
+```
+173.20.4.117 ceph117
+173.20.4.118 ceph118
+173.20.4.119 ceph119
+```
+
+### 3.配置master节点免密钥登陆节点
+
+选择一台部署主机，这里选择ceph117，在root用户下开启SSH互信操作
+
+```
+在部署节点上， 使用这个命令ssh-keygen   一路回车 ，生成公钥。
+然后通过命令ssh-copy-id -i  ~/.ssh/id_rsa.pub {hostname} 把公钥复制到部署需要设计的主机上。
+ssh hostname测试是否成功。
+```
+
+### 4.设置时间同步
+
+使用chrony或者ntp, 这里以chrony为例
+
+```
+yum install chrony
+systemctl enable chrony
+server x.x.x.x iburst    # ntp server
+local  stratum 8   # 如果本机做为server
+allow x.x.x.0/24   # 做为ntp server时候允许客户端访问的网络
+
+ntpdate -u 173.20.4.117  #客户端同步server的时间
+```
+
+[chrony部署参考连接](https://www.jianshu.com/p/7b2dd5813f92)
+
+### 5.内核参数配置确认
+
+确认内核参数已经添加下列条目，使用“sysctl -p”即可查看，如果没有出现下列条目，就需要手动配置：
+
+```
+net.ipv4.tcp_timestamps=1
+net.ipv4.tcp_keepalive_time = 1200
+net.ipv4.ip_local_port_range = 1024 65000
+net.ipv4.tcp_max_tw_buckets = 5000
+net.ipv4.tcp_max_syn_backlog = 8192
+net.ipv4.tcp_syncookies = 1
+net.ipv4.tcp_tw_reuse = 1
+net.ipv4.tcp_tw_recycle = 1
```

Comment:
perhaps this parameter will be dangerous in some use case,better to remove it for now.